.eslintrc.json

```diff
@@ -1,12 +1,18 @@
 {
   "extends": [
-    "eslint:recommended",
-    "plugin:prettier/recommended",
     "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:json/recommended"
   ],
-  "parser": "babel-eslint",
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true // Allows for the parsing of JSX
+    }
+  },
```

Comment:
Do we need to start using typescript?

I feel that it isn't really necessary here.

Reply:
agreed to use it, since blog is on TS